src/main/java/org/openrewrite/java/migrate/BeansXmlNamespace.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.openrewrite.*;
+import org.openrewrite.xml.ChangeTagAttribute;
+import org.openrewrite.xml.XPathMatcher;
+import org.openrewrite.xml.XmlVisitor;
+import org.openrewrite.xml.tree.Xml;
+
+@Value
+@EqualsAndHashCode(callSuper = true)
+public class BeansXmlNamespace extends Recipe {
+
+    private static final XPathMatcher BEANS_MATCHER = new XPathMatcher("/beans");
+    private static final String NS_SUN = "http://java.sun.com/xml/ns/javaee";
+    private static final String NS_JCP = "http://xmlns.jcp.org/xml/ns/javaee";
+    private static final String SUN_SCHEMA_LOCATION = "http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/beans_1_0.xsd";
+    private static final String JCP_SCHEMA_LOCATION = "http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd";
+    private static final String DISABLE_VALIDATING = "org.jboss.weld.xml.disableValidating";
+
+    @Override
+    public String getDisplayName() {
+        return "Check valid namespace and schema location in the`beans.xml` file ";
```

Comment:
I'm not sure the display name and description fully match what the recipe now does, so I'll propose two changes for you to consider, adapt & adopt or reject.
```suggestion
        return "Change `beans.xml` `schemaLocation` to match XML namespace";
```